std/mmfile.d

```diff
@@ -242,7 +242,7 @@ class MmFile
 
             int hi = cast(int)(size >> 32);
             hFileMap = CreateFileMappingW(hFile, null, flProtect,
-                    hi, cast(uint) size, null);
+                    hi, cast(uint) size, cast(LPCWSTR) null);
```

Comment:
Why is it necessary to cast `null`?

Reply:
It's not necessary, it's just a reminder in case someone wants to add the parameter.

Reply:
I don't get it, but OK.

Maybe subjective, but I don't like using `cast` unless it's strictly necessary, because of how powerful it is and how many kinds of things it can do. So perhaps `LPCWSTR(null)` or `LPCWSTR.init` or `/*LPCWSTR*/ null`.

Reply:
I understand, I'll yank it.